sui-execution/latest/sui-adapter/src/execution_engine.rs

```diff
@@ -714,6 +713,20 @@ mod checked {
                     trace_builder_opt,
                 )
             }
+            TransactionKind::ProgrammableSystemTransaction(pt) => {
+                programmable_transactions::execution::execute::<execution_mode::System>(
```

Comment:
What is different in this mode? Or what is the change here exactly?

Reply:
ability to call private / non-entry functions (i.e. the settlement code).

Reply:
So the only change was the mode?